Libraries/TranscriptConverter/Converter.cs

```diff
@@ -148,57 +148,59 @@ private static void AddAssertions(JToken token, ICollection<string> assertionsLi
         }
 
         /// <summary>
-        /// Checks if a string is a GUID value.
+        /// Checks if a string is a Base64 image.
         /// </summary>
-        /// <param name="guid">The string to check.</param>
-        /// <returns>True if the string is a GUID, otherwise, returns false.</returns>
-        private static bool IsGuid(string guid)
+        /// <param name="base64">The string to check.</param>
+        /// <returns>True if the string is a Base64 image, otherwise, returns false.</returns>
+        private static bool IsBase64Image(string base64)
         {
-            var guidMatch = Regex.Match(
-                guid,
-                @"([a-z0-9]{8}[-][a-z0-9]{4}[-][a-z0-9]{4}[-][a-z0-9]{4}[-][a-z0-9]{12})",
+            var base64Match = Regex.Match(
+                base64,
+                @"^data:image\/png;base64,",
                 RegexOptions.IgnoreCase);
-            return guidMatch.Success;
+            return base64Match.Success;
         }
 
         /// <summary>
-        /// Checks if a string is an ID value.
+        /// Checks if a property is an ID value.
         /// </summary>
-        /// <param name="id">The string to check.</param>
+        /// <param name="property">The ID property name to check.</param>
+        /// <param name="value">The ID property value to check.</param>
         /// <returns>True if the string is an ID, otherwise, returns false.</returns>
-        private static bool IsId(string id)
+        private static bool IsId(string property, string value)
         {
-            var idMatch = Regex.Match(
-                id,
-                @"([a-z0-9]{23})",
+            var specialCharactersMatch = Regex.Match(value, @"\r|\n|\t");
+
+            var guidMatch = Regex.Match(
+                value,
+                @"^([a-z0-9]{8}[-][a-z0-9]{4}[-][a-z0-9]{4}[-][a-z0-9]{4}[-][a-z0-9]{12})$",
                 RegexOptions.IgnoreCase);
-            return idMatch.Success;
+
+            var idPropertyMatch = Regex.Match(
+                property,
+                @"^id|id$",
+                RegexOptions.IgnoreCase);
+
+            return !specialCharactersMatch.Success && (guidMatch.Success || idPropertyMatch.Success);
         }
 
         /// <summary>
-        /// Checks if a string is a service url value.
+        /// Checks if a string is an url value.

```

Comment:
This is internal but just for ref, it is a URL, not an URL (https://whatis.techtarget.com/feature/Which-is-correct-a-URL-or-an-URL :-) )